include/zephyr/bluetooth/mesh/cfg_cli.h

```diff
@@ -37,9 +37,72 @@ struct bt_mesh_cfg_cli_cb {
 	 *  @param mod_id      The model ID within the element.
 	 */
 	void (*mod_sub_status)(struct bt_mesh_cfg_cli *cli, uint16_t addr,
-			     uint8_t status, uint16_t elem_addr,
-			     uint16_t sub_addr, uint32_t mod_id);
+			uint8_t status, uint16_t elem_addr,
+			uint16_t sub_addr, uint32_t mod_id);
```

Comment:
I don't always trust what the web browser is showing wrt whitespace, indentation and alignment. It looked like the change you're making might be wrong, which is why I asked you to double check it. I now checked out the PR locally and opened the file in my editor, and indeed, this change doesn't look right. Split lines are supposed to be aligned so that their beginning consists of N tabs + 0-7 spaces (assuming tab-width of 8) so that the start of the split line begins right after the opening parenthesis of the first line.

So it seems you can just drop this commit from the PR.